src/App/Range.cpp

```diff
@@ -197,9 +197,7 @@ int App::validColumn(const std::string &colstr)
   *
   * This function will throw an exception if the specified \a address is invalid.
   *
-  * @param address Address to parse.
-  * @param row     Reference to integer where row position is stored.
-  * @param col     Reference to integer where col position is stored.
+  * @param strAddress Address to parse.
   *
```

Comment:
That's what doxygen complains about these parameters are nowhere to be seen in the actual declaration/definition and address has been renamed to strAddress. The code hasn't been touched. The doxygen comments are just reflecting what's implemented.